doc/metadata.txt

```diff
@@ -5074,6 +5081,14 @@ This option specifies the number of OSD pool placement groups (`pg_num`) to use
 when creating a missing OSD pool.
 ```
 
+```{config:option} cephfs.osd_pool_size storage-cephfs-pool-conf
+:defaultdesc: "`3`"
+:shortdesc: "Number of RADOS object replicas. Set to 1 for no replication."
+:type: "string"
+This option specifies the number of OSD pool replicas to use
+when creating an OSD pool.
```

Comment:
Why have a long description for `cephfs` and not for `ceph`? Also, is this really something that needs to be done at creation time only?

Reply:
Added it now.